helios-tools/src/main/java/com/spotify/helios/cli/CliConfig.java

```diff
@@ -157,6 +157,11 @@ public static CliConfig fromEnvVar(Map<String, Object> config) throws URISyntaxE
 
     // TODO (dxia) Remove DEPRECATED_SITE_SCHEME after a period of time
     final String scheme = uri.getScheme();
+    if (scheme == null) {
```

Comment:
Maybe use `isNullOrEmpty` here? Best practice is to treat null/empty strings the same.